cmd/manager/main.go

```diff
@@ -101,9 +157,46 @@ func main() {
 	cl := mgr.GetClient()
 	catalogClient := catalogclient.New(cl, cache.NewFilesystemCache(cachePath, &http.Client{Timeout: 10 * time.Second}))
 
+	cfgGetter, err := helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), helmclient.StorageNamespaceMapper(func(o client.Object) (string, error) {
+		return systemNamespace, nil
+	}))
+	if err != nil {
+		setupLog.Error(err, "unable to config for creating helm client")
+		os.Exit(1)
+	}
+
+	acg, err := helmclient.NewActionClientGetter(cfgGetter)
+	if err != nil {
+		setupLog.Error(err, "unable to create helm client")
+		os.Exit(1)
+	}
+
+	unpacker, err := source.NewDefaultUnpacker(mgr, systemNamespace, unpackImage, (*x509.CertPool)(nil))
```

Comment:
IIRC the default unpacker has more source types than we are likely concerned with for now. Does it make sense to create a slimmed down unpacker that only has the image source for unpacking?

Reply:
The other unpacker sources are going to be removed from Rukpak too. So the change would be in there. This was brought up before, but we then decided to do the bare minimal changes in Rukpak and o-c and not introduce a new unpacker. We can create a follow up to handle this in Rukpak.